modules/rational-files-management.el

```diff
@@ -0,0 +1,19 @@
+;;; rational-files-management.el -*- lexical-binding: t; -*-
+
+(require 'dired)
+
+;; Always delete and copy recursively
+(setq dired-recursive-deletes 'always)
+(setq dired-recursive-copies 'always)
+
+;; Create destinations directories if needed
+(setq dired-create-destination-dirs 'always)
+
+;; Refresh buffers automatically
+(add-hook 'dired-mode-hook 'auto-revert-mode)
+
+;; Display directories first
+(setq dired-listing-switches "-vAlahF --group-directories-first")
```

Comment:
We might need to add a check for whether we're on Windows and use a different set of params, but I'm not sure what Dired actually uses for dir listings on Windows...

Reply:
The -A option suppresses listing the . and .. entries while -a re-introduces them again, so it should be either -A or -a, not both options at the same time.